test/spec/features/popup-menu/ReplaceMenuProviderSpec.js

```diff
@@ -49,6 +50,144 @@ describe('features/popup-menu - replace menu provider', function() {
     });
   };
 
+  describe('data object - collection marker', function() {
```

Comment:
As mentioned earlier today, we should split up the UI specific tests from the ones that validate the API (`Modeling#...`) usage and `commandStack.changed` semantics.